rosdep/base.yaml

```diff
@@ -7652,20 +7684,23 @@ postgresql-client:
   debian: [postgresql-client]
   fedora: [postgresql]
   gentoo: [dev-db/postgresql]
+  nixos: [postgresql]
   ubuntu: [postgresql-client]
 postgresql-postgis:
   debian:
     buster: [postgresql-11-postgis-2.5, postgresql-contrib, postgresql-server-dev-all]
     stretch: [postgresql-9.6-postgis-2.3, postgresql-contrib, postgresql-server-dev-all]
   fedora: [postgis]
   gentoo: [dev-db/postgis, =dev-db/postgresql-9*]
+  nixos: [postgresqlPackages.postgis]
```

Comment:
There is no `postgresqlPackages.postgis` in the nixos index.
However, there present pinned versions like [postgresql16Packages.postgis](https://search.nixos.org/packages?channel=24.05&from=0&size=50&sort=relevance&type=packages&query=postgis) or [postgresql15Packages.postgis](https://search.nixos.org/packages?channel=24.05&from=0&size=50&sort=relevance&type=packages&query=postgis)

I think we need to refer to one of the pinned versions.

Reply:
It's an alias like `python3Packages`:


```console
[jkolb@nixos:~/nix-ros-overlay]$ nix build nixpkgs#postgresqlPackages.postgis

[jkolb@nixos:~/nix-ros-overlay]$ ls -l result
lrwxrwxrwx 1 jkolb users 57 Oct  3 09:37 result -> /nix/store/366llrbbdjw0dim6xh4m01dkncg0mmv1-postgis-3.4.3
```

Reply:
Ok, then. Thanks for the clarification.